yt/geometry/coordinates/cartesian_coordinates.py

```diff
@@ -119,6 +157,57 @@ def pixelize(self, dimension, data_source, field, bounds, size,
             return self._oblique_pixelize(data_source, field, bounds, size,
                                           antialias)
 
+
+    def line_plot(self, field, start_point, end_point, npoints):
```

Comment:
I'm probably going to use this for implementing fixed resolution rays in region expression after this is merged in, so I think a name change is in order, since that use won't be for plotting. Something like `line_pixelize` or `pixelize_line` instead. I'm happy to take care of that myself when I do that, however, so no need to change it now.